RELEASE.md

```diff
@@ -1,78 +1,234 @@
-    * e2e-test-runner
-    * e2e-test-cfssl
-    * e2e-test-echo
-    * e2e-test-fastcgi-helloserver
-    * e2e-test-httpbin
+    - Make that snippet look like this ;
+      ```
+      data:
+        http-snippet:|
+          server{
+          listen 2443;
+          return 308 https://$host$request_uri;
+          }
+      ```
 
-3. Update references to e2e-test-runner image:
+### f. Edit the changelog
+  [Changelog.md](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md) 
+    - Add the PRs merged after previous release
 
-* [e2e-image](https://github.com/kubernetes/ingress-nginx/blob/main/test/e2e-image/Dockerfile#L1)
-* [run-in-docker.sh](https://github.com/kubernetes/ingress-nginx/blob/ff60aa9e2b5377db1544091b98f475a90a630297/build/run-in-docker.sh#L37)
+### g. Edit the Documentation:
+    - Update the version in [docs/deploy/index.md](docs/deploy/index.md)
+    - Update Supported versions in the Support Versions table in the README.md 
 
-4. Prepare for a new release:
+### h. Edit stable.txt
 
-* Change [TAG](https://github.com/kubernetes/ingress-nginx/blob/main/TAG#L1)
-* Open pull request
-* Merge
-* [Wait for cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+- Edit the [stable.txt](stable.txt) file(if applicable), in the root of the repo, to reflect the release to be created
 
-4a. Promote images:
+### i. Edit ghpages
 
-* Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
-  * controller
+- Edit the ghpages file as needed
 
-5. Release helm chart:
+## 5. RELEASE new version
 
-* Open pull request updating [Chart.yaml](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/Chart.yaml#L3-L4)
-* Merge
-* [New helm chart is available](https://github.com/kubernetes/ingress-nginx/blob/main/.github/workflows/main.yaml#L47-L68)
+### a. Create PR
 
-6. New release:
+- Open PR for releasing the new version of the Ingress-Nginx-Controller ;
+  - Look at this PR for how it was done before [example PR](https://github.com/kubernetes/ingress-nginx/pull/7490)
+  - Create a PR 
 
-* Update static scripts:
-    * [generate-deploy-scripts.sh](https://github.com/kubernetes/ingress-nginx/blob/main/hack/generate-deploy-scripts.sh)
-    * Open pull request with the updates
-    * Merge
+### b. Merge
 
-* Update Changelog and Documentation:
-    * Open pull request updating [Changelog.md](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md)
-    * Update the version in [docs/deploy/index.md](docs/deploy/index.md)
-    * Update Supported versions in the Support Versions table in the README.md 
-    * Merge
-      
-7. Github release
+- Merge should produce manifests as well as chart
+- Check
+ - `helm repo update`
+ - `helm search repo ingress-nginx`
```

Comment:
Still need the Github Release as the last step

Reply:
got it now. changing

Reply:
Finally made ghpages as step 6

Reply:
The GitHub release not page. Aka what @rikatz does